templates/tfengine/components/audit/variables.tf

```diff
@@ -15,13 +15,55 @@
 {{if eq .parent_type "organization" -}}
 variable "org_id" {
   type = string
+  description = "ID of the organization to apply the configuration."
+  validation {
+    condition     = can(regex("^[0-9]{8,25}$", var.org_id))
+    error_message = "The org_id must be valid. Should have only numeric values with a length between 8 and 25 digits. See https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy to know how to get your organization id."
+  }
 }
 {{- else}}
 variable "folder" {
   type = string
+  description = "ID of the folder to apply the configuration."
+  validation {
+    condition     = can(regex("^[0-9]{8,25}$", var.folder))
+    error_message = "The folder must be valid. Should have only numeric values with a length between 8 and 25 digits. See https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy to know how to get your folder id."
+  }
 }
 {{- end}}
 
 variable "auditors_group" {
   type = string
+  description = "This group will be granted viewer access to the audit log dataset and bucket as well as security reviewer permission on the root resource specified."
+}
+
+variable "filter" {
+  type = list(string)
```

Comment:
It seems like it should just be `string`, not `list(string)`.

Reply:
That's correct. Fixed. Thanks